pkg/client/events/replay_client.go

```diff
@@ -27,19 +27,17 @@ const (
 	// eventsBytesRetryDelay is the delay between retry attempts when the events
 	// bytes observable returns an error.
 	eventsBytesRetryDelay = time.Second
-
-	// replayObsCacheBufferSize is the replay buffer size of the
-	// replayObsCache replay observable which is used to cache the replay
-	// observable that is notified of new events.
-	// It, replayObsCache, is updated with a new "active" observable when a new
-	// events query subscription is created, for example, after a non-persistent
-	// connection error.
-	replayObsCacheBufferSize = 1
 )
 
 // Enforce the EventsReplayClient interface is implemented by the replayClient type.
 var _ client.EventsReplayClient[any] = (*replayClient[any])(nil)
 
+// Define connection state for tracking and logging transitions
```

Comment:
I'm not complete sure what state this is referring to.

The full node?

A single endpoint?

A single request?

Something else?